build.sbt

```diff
@@ -42,8 +42,9 @@ libraryDependencies ++= {
 scalacOptions in ThisBuild ++= Seq("-language:postfixOps",
                                    "-language:implicitConversions",
                                    "-language:existentials",
+                                   "-Xfatal-warnings",
                                    "-feature",
-                                   "-deprecation")
+                                   "-deprecation:false")
```

Comment:
BTW, this is soooo cool, I didn't know about it! Thanks for teaching me something new ❤️